implicits/src/com/rallyhealth/weepickle/v1/implicits/DefaultTos.scala

```diff
@@ -48,6 +49,7 @@ trait DefaultTos extends com.rallyhealth.weepickle.v1.core.Types with Generated
 
   implicit val ToInt: To[Int] = new NumericTo[Int] {
     override def expectedMsg = "expected number"
+    override def visitString(cs: CharSequence) = cs.toString.toInt
```

Comment:
What are the downsides of doing this type coercion for non-XML types, if any?

Reply:
It is additive, so it does not introduce any new cases where things that used to work now break. It certainly add cases where things that used to break now work (like `"-666"` as a string above). So I guess the downside is reduced static type safety relative to that, i.e., if you wanted some warning that you really ought to represent numbers as numbers instead of strings, you aren't going to get it anymore. And some strings will parse as ints and others won't, so all type conversion errors become runtime errors. HOWEVER, in nearly all cases (e.g., JSON parser) what `visit*` method to call is already inferred at runtime, so it isn't exactly "static" today, so there isn't much static type safety being giving up IMO. 

Also, as inferred in the PR description, adding this for boolean, int, short, and byte makes us _more_ consistent with what is already being done for double and float. If we want to fall back on maximum static safety, `ToDouble` and `ToFloat` should be modified to remove their implementations of `visitString`.